### PR TITLE
first stab at #119

### LINKED ISSUE #119: clarify PSA scope and applicability
Eliot:

> I would like some statement, without it turning into an advertisement, about who and what PSA is.  Is this really just an ARM initiative or is it broader?  Some clarity would help, so that implementors understand applicability.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix #119 

suggestions welcome :-)

### CODE REVIEW

**Comment by adrianlshaw** [comment]
I think it looks fine. It could be better integrated at the beginning like so:

> The Platform Security Architecture (PSA) is a family of hardware and firmware security specifications maintained by  {{PSACertified}}. PSA Certified is an independent collaborative effort maintained by companies from multiple sectors including evaluation laboratories, semiconductor IP and security consultancy to build security best practices into devices at scale.
>  ...

**Comment by thomas-fossati** [comment]
> I think it looks fine. It could be better integrated at the beginning like so:
> 
> > The Platform Security Architecture (PSA) is a family of hardware and firmware security specifications maintained by  {{PSACertified}}. PSA Certified is an independent collaborative effort maintained by companies from multiple sectors including evaluation laboratories, semiconductor IP and security consultancy to build security best practices into devices at scale.
> > Devices that are PSA compliant ...

I like it, thanks, Adrian!  The only "problem" is it drops _"[...] as well as open-source reference implementations"_ which I think is a pretty strong point.

**Comment by hannestschofenig** [comment]
I wonder whether it is really correct to call it a "independent collaborative effort". It is an Arm-effort but the companies doing the certifications are third parties.

**Comment by adrianlshaw** [comment]
> I like it, thanks, Adrian! The only "problem" is it drops "[...] as well as open-source reference implementations" which I think is a pretty strong point of the PSA proposition.

I share the same concern.

> I wonder whether it is really correct to call it a "independent collaborative effort". It is an Arm-effort but the companies doing the certifications are third parties.

Correct or not, I think "Independent collaborative effort" comes from the description on the official website https://www.psacertified.org/what-is-psa-certified/founding-members/

File: draft-tschofenig-rats-psa-token.md [hunk 1]
```diff
@@ -197,6 +196,8 @@ encoded and cryptographically protected.
 Further details on concepts expressed below can be found in the PSA Security
 Model documentation {{PSA-SM}}.
 
+{{PSACertified}} is an independent collaborative effort maintained by companies from multiple sectors including evaluation laboratories, semiconductor IP and security consultancy to build security best practices into devices at scale.
```
Comment (hannestschofenig) [inline draft-tschofenig-rats-psa-token.md:199]:
```suggestion
The Platform Security Architecture (PSA) is a set of security specifications accompanied by reference implementations, which are combined in a security certification scheme. The specifications as well as information about the security certification is maintained by {{PSACertified}}. While the security specifications have been published by Arm, the certification scheme called "PSA Certified" as well as the reference implementations are a collaborative effort by companies from multiple sectors, including evaluation laboratories, semiconductor IP and security consultancy, to improve security of IoT devices.
```

**Review by SimonFrost-Arm — COMMENTED** [review]
LGTM. One small suggestion inline to help link together PSA & PSA Certified

File: draft-tschofenig-rats-psa-token.md [hunk 2]
```diff
@@ -161,16 +161,20 @@ entity:
 
 --- abstract
 
-The Platform Security Architecture (PSA) is a family of hardware and firmware
-security specifications, as well as open-source reference implementations, to
-help device makers and chip manufacturers build best-practice security into
-products. Devices that are PSA compliant can produce attestation tokens
+The Platform Security Architecture (PSA) is a set of hardware and firmware
+specifications, backed by reference implementations and a security
+certification program.  The security specifications have been published by Arm,
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply draft-tschofenig-rats-psa-token.md]: done in [5423409](https://github.com/thomas-fossati/draft-psa-token/pull/123/commits/5423409d6aa36b91f78e9e43a8d1e058820a968f)

**Review by SimonFrost-Arm — APPROVED** [review]
LGTM

**Pull request merged**